keyboards/mwstudio/mw80/mw80.c

```diff
@@ -0,0 +1,43 @@
+/* Copyright 2022 TW59420 <https://github.com/TW59420>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "mw80.h"
+
+#ifdef RGB_MATRIX_ENABLE
+led_config_t g_led_config = { {
+
```

Comment:
This section of the LED config should not be blank, and should match matrix width/height.

Reply:
no led in matrix, only mood lights

Reply:
It still can't be blank.